java/pom.xml

```diff
@@ -105,6 +107,11 @@
         <artifactId>janino</artifactId>
         <version>${janino.version}</version>
       </dependency>
+      <dependency>
```

Comment:
could we move this dependency version info into fury-extensions module? we try to keep this pom simplistic